src/main/resources/application.properties

```diff
@@ -9,7 +9,8 @@
 
 # Determines the type of credential used for Key Vault Access
 # Valid Values = MSI || CLI
-# Linux: export He_EnvironmentFlag=CLI
+
+# Linux: set He_EnvironmentFlag=CLI
```

Comment:
both work, any particular reason why set is preferable over export?

Reply:
No specific preference, we can leave it to export